app/main.development.js

```diff
@@ -12,6 +12,9 @@ if (process.env.NODE_ENV === 'production') {
 
 if (process.env.NODE_ENV === 'development') {
   require('electron-debug')(); // eslint-disable-line global-require
+  const path = require('path'); // eslint-disable-line
+  const p = path.join(__dirname, '..', 'static', 'node_modules'); // eslint-disable-line
+  require('module').globalPaths.push(p); // eslint-disable-line
```

Comment:
This was supposed to fix the "module not found" issue in `dev` mode (`npm run dev`) when a module is in the `static/node_modules` folder? Still having a problem with that on my end